credits.txt

```diff
@@ -1,6 +1,6 @@
-Welcome to Endless Sky!
+Welcome to Endless Sky Delta!
 
-version 0.10.10-delta
+version 0.10.11-alpha
```

Comment:
```suggestion
version 0.10.11-delta
```
Historically, the theory has been to identify things as being part of the next release. Which got missed in the last major release cycle for us, from the looks of it, so that's a good catch. Upstream does so via the 0.10.11-alpha identifier (all the continuous builds will have the -alpha, with it being removed from the actual release of that version number, then re-appended when it moves up to the next one. The theory being it shows which release the code is being worked on.

We've been using the "delta" on the end of that as ESD is basically considered an alternative release/stream (by me, anyway). But maybe we should numericalize the continuous releases instead of using the alpha designation.

Having all the continuous ones until release be 0.10.10.5-delta for all the current development; then it would be0.10.11.0-delta for the next release, folowed by 0.10.11.5-delta, etc. That keeps things in numeric order, and avoids messing with alpha terminology that might confuse with the Delta term.